### ISSUE TITLE
v3.0.1R4 needs motor kill if upside down for more than two seconds

### ISSUE THREAD

**Issue by thomasbutler** [MegaPirateNG/ardupilot-mpng#127]
v3.0.1R4 works greate, but needs motor kill if upside down for more than five seconds (i.e., crashed). I think I saw a two second timeout/kill was added to v3.1 of ArduCopter.  This would be a valuable addition to v3.0.1 before the leap to v3.1.  Is there going to be another release for 3.0.1 that might perhaps have this?

**Comment by smurfy** [comment]
For 3.0.1 i would say, sorry but no. 
@SirAlex is still working on 3.1 but time is always a problem. please be patient.
It makes no sense backporting that feature to 3.0.1 while 3.1 is almost working.

**Comment by thomasbutler** [comment]
Thanks for getting back to me!  I honestly didn't expect it to be ported back. As a software developer myself, one of the things I hate to do is back-port new features.   However, I might do it for my own copy of R4 and post the result.   I did a patch of LED output to pin 32 when external  COPTER_LEDS funtionality wasn't ready.  I crashed my copter on a windy day, and it landed on a porch roof upside down.  I was in a panic that the ESCs would burn out, but they didn't after I calmed down and turned off loiter; the auto shutoff after ten seconds would have been good (two is a bit short).  

Good luck and no hurry!
